firstvoices/firstvoices/settings.py

```diff
@@ -230,3 +230,17 @@
         ),
     ],
 )
+
+
+# File hosting on AWS S3
+# See: https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html
+DEFAULT_FILE_STORAGE = "storages.backends.s3boto3.S3Boto3Storage"
+AWS_ACCESS_KEY_ID = os.getenv("AWS_ACCESS_KEY_ID")
+AWS_SECRET_ACCESS_KEY = os.getenv("AWS_SECRET_ACCESS_KEY")
+AWS_STORAGE_BUCKET_NAME = os.getenv("MEDIA_UPLOAD_S3_BUCKET")
+AWS_S3_REGION_NAME = os.getenv("MEDIA_UPLOAD_S3_REGION")
```

Comment:
These are the same env variables we were using for v2, so you may already have these set up for local dev, possibly in an env file in your fv-web-ui repo